pkgs/development/python-modules/fava-investor/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  stdenv,
+  beancount,
+  click,
+  click-aliases,
+  fava,
+  hatch-vcs,
+  hatchling,
+  packaging,
+  pytestCheckHook,
+  python-dateutil,
+  tabulate,
+  yfinance,
+}:
+buildPythonPackage rec {
+  pname = "fava_investor";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "redstreet";
+    repo = "fava_investor";
+    tag = version;
+    hash = "sha256-WuXbZcia0n9SoiCSB2SkMUjBHsMOA0gCIf9ZEU9pTPA=";
+  };
+
+  build-system = [
+    hatchling
```

Comment:
Looks like the project is using [setup.py](https://github.com/redstreet/fava_investor/blob/1.0.1/setup.py), shouldn't setuptools be used here?